dvc/utils/ls_colors.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
Feels like this code won't ever be used by anyone else. So maybe let's create `dvc/repo/ls` directory and place this and colorize in there instead to keep it organized?

Reply:
Better: see https://github.com/iterative/dvc/pull/3351/files#r380398252